### ISSUE TITLE
A QUESTION ABOUT THE VERSION OF BIOBOX

### ISSUE THREAD

**Issue by jiaming-sheep** [aaayushg/generative_IDPs#5]
Hello, I am starting a new project with code your uploaded. But confusingly, when I downloaded and installed the biobox, errors are reported. With a few days of trying, I found the version of biobox is  important . The high version of biobox can not run with the code you uploaded, but the low version can not be installed. So could you tell me which version of biobox you usually use? It may be very meaningful for me!

**Comment by aaayushg** [comment]
hey could you try [biobox v1.0.0](https://github.com/Degiacomi-Lab/biobox/releases/tag/v1.0.0) https://github.com/Degiacomi-Lab/biobox/releases or 1 version up?
This release is close to the timeline I did this work. Hopefully, it will resolve the issue.

**Comment by jiaming-sheep** [comment]
Oh, thanks for your answer. But it still reports errors. It may be a parameter error？
![BD932BD291946F4D4DACCFB3956CB4A5](https://github.com/user-attachments/assets/0fad6e37-f05a-4e87-99eb-adf9ab496fbb)
![524D9C7942E8A79363B2C49147C0C641](https://github.com/user-attachments/assets/5e7494c0-1ae7-4b50-b998-01db1f22fe20)

**Comment by aaayushg** [comment]
Looks like there are only 1000 conformations (indices) and you are trying to access 95000 indices.

**Comment by jiaming-sheep** [comment]
So it means that the training data should be suitable with the parameter setting? Should I rewrite the parameter setting or increase the amount of training data？

**Comment by aaayushg** [comment]
yeah as you are using training script. you need indices matching with number of training conformations. 
To generate new conformation (which uses trained model) there is another script and you can define as many indices you want.
